### PR TITLE
Suggested edits to 5.4

### PR DESCRIPTION
**Pull request by cansavvy**
Looks like great adds! There's some punctuation things I have here, but I also wonder if a few of these points could be further explained. I'll put comments where I mean.

### CODE REVIEW

File: 05-promoting_diversity.Rmd [hunk 1]
```diff
@@ -49,10 +49,10 @@ Furthermore, analyses of university faculty and students demonstrate that both w
 ## Underrepresentation in clinical trials
 
 
-Beyond underrepresentation of certain populations within research teams, there is also a lack of representation of various populations in clinical trials (in particular people of color, woman, and with limited data collected about LGBTQ+ individuals [@chen_approach_2019])  [@clark_increasing_2019]. This is particularity true for cancer clinical trial studies [@nazha_enrollment_2019]. 
+Beyond underrepresentation of certain populations within research teams, there is also a lack of representation of various populations in clinical trials (in particular people of color, woman, and with limited data collected about LGBTQ+ individuals [@chen_approach_2019])  [@clark_increasing_2019]. This is particularity true for cancer clinical trial studies [@nazha_enrollment_2019].
```
Comment (cansavvy) [inline 05-promoting_diversity.Rmd:52]:
```suggestion
Beyond underrepresentation of certain populations within research teams, there is also a lack of representation of various populations in clinical trials (in particular people of color, women, and with limited data collected about LGBTQ+ individuals [@chen_approach_2019])  [@clark_increasing_2019]. 
This is particularity true for cancer clinical trial studies [@nazha_enrollment_2019].
```

I left some notes about this sentence in basecamp.

File: 05-promoting_diversity.Rmd [hunk 2]
```diff
@@ -148,30 +149,33 @@ Recent projects in which participants are recruited through community engagement
...
 
 3) More promotion of community engagement
 
-Community stakeholders can help to assess if recruitment information is culturally appropriate and can help with creating more equitable recruitment strategies. 
+Community stakeholders can help assess if recruitment information is culturally appropriate and can help create more equitable recruitment strategies.
 
 4) More inclusive research teams
 
-Individuals have been shown to be more trusting when research teams include members that are similar to themselves. [@griffith_determinants_2020]
+Individuals have been shown to be more trusting when research teams include members that are similar to themselves [@griffith_determinants_2020].
```
Comment (cansavvy) [inline 05-promoting_diversity.Rmd:166]: More similar in what ways? Can we be more specific here? Background/SES/race/gender?

File: 05-promoting_diversity.Rmd [hunk 3]
```diff
@@ -148,30 +149,33 @@ Recent projects in which participants are recruited through community engagement
...
 
 5) More inclusive practices
 
-When obtaining information about gender, researchers should collect more informative information such as the model proposed in @chen_approach_2019. Similarly, more detailed information about ancestry could also lead to more informative findings about specific populations. 
+When obtaining information about gender, researchers should collect more informative information such as the model proposed in @chen_approach_2019.
+Similarly, more detailed information about ancestry could also lead to more informative findings about specific populations.
```
Comment (cansavvy) [inline 05-promoting_diversity.Rmd:171]: This feels like a separate point from the one before it. Can more info be added about both of these points?

File: 05-promoting_diversity.Rmd [hunk 4]
```diff
@@ -148,30 +149,33 @@ Recent projects in which participants are recruited through community engagement
...
+Investigators should consider how to support caregivers who have important obligations to tend to their family members or others, for example when possible, perhaps home visits would be feasible.
+Other strategies include providing flexible hours for trial visits and support for transportation, such as payment for a rideshare service or taxi [@ACRP_representation_2020]. Researchers should also consider providing patients with a cell phone to improve communication [@clark_increasing_2019].
 
-2) More promotion of health care access in general 
+2) More promotion of health care access in general
 
-Individuals have been shown to be more likely to participate when advised by a health care provider.[@clark_increasing_2019] Increasing health care system access in general could therefore improve clinical trial participation.
+Individuals are more likely to participate when advised by a health care provider [@clark_increasing_2019].
+Increasing health care access in general could therefore improve clinical trial participation.
```
Comment (cansavvy) [inline 05-promoting_diversity.Rmd:158]:
```suggestion
Increasing health care access in general could improve clinical trial participation.
```

File: 05-promoting_diversity.Rmd [hunk 5]
```diff
@@ -148,30 +149,33 @@ Recent projects in which participants are recruited through community engagement
 1) Provide adequate financial, logistical, and time support
 
 Researchers should consider how participants can cope with healthcare needs following participation in a study.
-Investigators should consider how to support caregivers who have important obligations to tend to their family members or others, for example when possible, perhaps home visits would be feasible. Other strategies include providing flexible hours for trial visits and support for transportation, such as payment for a rideshare service or taxi.[@ACRP_representation_2020] Researchers should also consider providing patients with a cell phone to improve communication.[@clark_increasing_2019]
+Investigators should consider how to support caregivers who have important obligations to tend to their family members or others, for example when possible, perhaps home visits would be feasible.
```
Comment (cansavvy) [inline 05-promoting_diversity.Rmd:152]: I'm a bit confused on the flow of this paragraph and how this sentence is/is not related to the one before it. Feels a little like its a separate point?

**Comment by carriewright11** [comment]
Thanks for these! some of these got fixed or altered with your other suggestions. Really appreciate your help!

**Pull request closed without merging**